common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1101,14 +1110,14 @@
                 this.captionControlEl
                     .addClass('is-active')
                     .find('.control-text')
-                        .text(gettext('Hide closed captions'));
+                        .text(window.gettext('Hide closed captions'));
 
                 if (this.subtitlesEl.find('.current').text()) {
                     this.captionDisplayEl
                         .text(this.subtitlesEl.find('.current').text());
                 } else {
                     this.captionDisplayEl
-                        .text(gettext('(Caption will be displayed when you start playing the video.)'));
+                        .text(window.gettext('(Caption will be displayed when you start playing the video.)'));
```

Comment:
Nit: you could have left all of these as just gettext. I agree that the file should be consistent, so if you back out these changes, also change your new usages to just be gettext.